build.sbt

```diff
@@ -667,9 +666,6 @@ lazy val zincScripted = (projectMatrix in internalPath / "zinc-scripted")
   )
   .defaultAxes(VirtualAxis.jvm, VirtualAxis.scalaPartialVersion(scala212))
   .jvmPlatform(scalaVersions = List(scala212))
-  .configure(
-    _.dependsOn(compilerBridge210, compilerBridge211, compilerBridge212, compilerBridge213)
```

Comment:
This makes `zincScripted` depend on all compiler bridge JARs whereas the one above will only depend on the ones for Scala 2.12? I wonder if this was meant to create build order dependency so calling the scripted test would force rebuilding compiler bridges when we change the bridge imple. Could you check if that still works plz?